pkg/serviceaccount/claims.go

```diff
@@ -51,21 +53,33 @@ type kubernetes struct {
 	Pod       *ref             `json:"pod,omitempty"`
 	Secret    *ref             `json:"secret,omitempty"`
 	WarnAfter *jwt.NumericDate `json:"warnafter,omitempty"`
+
+	RequesterUserInfo userInfo `json:"requesterUserInfo,omitempty"`
 }
 
 type ref struct {
 	Name string `json:"name,omitempty"`
 	UID  string `json:"uid,omitempty"`
 }
 
-func Claims(sa core.ServiceAccount, pod *core.Pod, secret *core.Secret, expirationSeconds, warnafter int64, audience []string) (*jwt.Claims, interface{}) {
+type userInfo struct {
+	// The name that uniquely identifies this user among all active users.
+	Username string `json:"username,omitempty"`
```

Comment:
Can we think through the privacy implications (if any exist) of adding this field and including in things like audit logs?